File: app/src/main/java/br/com/victoramaral/githubdive/view/activities/SplashActivity.java
package br.com.victoramaral.githubdive.view.activities;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import java.util.Timer;
import java.util.TimerTask;

import br.com.victoramaral.githubdive.R;

public class SplashActivity extends AppCompatActivity {

    private Timer timer = new Timer();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        timer.schedule(new TimerTask() {
            @Override
            public void run() {
                jump();
            }
        }, 2000);
    }

    private void jump() {
        timer.cancel();
        startActivity(new Intent(SplashActivity.this,
                LoginActivity.class));
        finish();
    }
}

File: app/src/main/java/com/rafaelmfer/consultinggithub/mvvm/SplashActivity.kt
package com.rafaelmfer.consultinggithub.mvvm

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.rafaelmfer.consultinggithub.mvvm.view.HomeActivity


class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this@SplashActivity, HomeActivity::class.java)
        startActivity(intent)
        finish()
    }
}